reconcile/utils/ldap_client.py

```diff
@@ -27,11 +27,15 @@ def init_from_config():
     return init(serverUrl)
 
 
-def user_exists(username):
+def get_users():
     global _base_dn
 
     with init_from_config() as client:
-        result, _, _, _ = client.search(_base_dn,
-                                        f'(&(uid={username})' +
-                                        '(objectclass=person))')
-        return result
+        _, _, results, _ = client.search(_base_dn, '(&(objectclass=person))')
+        users = set()
+        for r in results:
+            dn = r['dn']
```

Comment:
Just extract the uid field here.
```suggestion
            uid = r['uid']
```

Reply:
thanks!